mmv1/products/redis/api.yaml

```diff
@@ -143,6 +143,125 @@ objects:
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits.
+          - !ruby/object:Api::Type::String
+            name: 'updateTime'
+            output: true
+            description: |
+              Output only. The time when the policy was last updated.
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits.
+          - !ruby/object:Api::Type::String
+            name: 'description'
+            description: |
+              Optional. Description of what this policy is for.
+              Create/Update methods return INVALID_ARGUMENT if the
+              length is greater than 512.
+          - !ruby/object:Api::Type::Array
+            name: 'weeklyMaintenanceWindow'
+            description: |
+              Optional. Maintenance window that is applied to resources covered by this policy.
+              Minimum 1. For the current version, the maximum number
+              of weekly_window is expected to be one.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: 'day'
+                  required: true
+                  description: |
+                    Required. The day of week that maintenance updates occur.
+
+                    - DAY_OF_WEEK_UNSPECIFIED: The day of the week is unspecified.
+                    - MONDAY: Monday
+                    - TUESDAY: Tuesday
+                    - WEDNESDAY: Wednesday
+                    - THURSDAY: Thursday
+                    - FRIDAY: Friday
+                    - SATURDAY: Saturday
+                    - SUNDAY: Sunday
+                  values:
+                    - :DAY_OF_WEEK_UNSPECIFIED
+                    - :MONDAY
+                    - :TUESDAY
+                    - :WEDNESDAY
+                    - :THURSDAY
+                    - :FRIDAY
+                    - :SATURDAY
+                    - :SUNDAY
+                - !ruby/object:Api::Type::String
+                  name: 'duration'
+                  output: true
+                  description: |
+                    Output only. Duration of the maintenance window.
+                    The current window is fixed at 1 hour.
+                    A duration in seconds with up to nine fractional digits,
+                    terminated by 's'. Example: "3.5s".
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'startTime'
+                  required: true
+                  allow_empty_object: true
+                  send_empty_value: true
+                  description: |
+                    Required. Start time of the window in UTC time.
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: 'hours'
+                      default_value: 0
```

Comment:
not needed, go integers default to 0 anyways

Reply:
A small note about this. The `authEnabled` property (not part of the changes here) also has `default_value` set, even though boolean zero-value is `false`. Perhaps something to address if you want the consistency at some point later.

Reply:
correct, there is no point to this. Thanks for pointing that out ^-^

Reply:
done